gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagActionReminderScheduler.java

```diff
@@ -86,18 +88,18 @@ public static class ReminderJob implements Job {
     @Override
     public void execute(JobExecutionContext context) {
       // Get properties from the trigger to create a dagAction
-      JobDataMap jobDataMap = context.getTrigger().getJobDataMap();
+      JobDataMap jobDataMap = context.getMergedJobDataMap();
       String flowName = jobDataMap.getString(ConfigurationKeys.FLOW_NAME_KEY);
       String flowGroup = jobDataMap.getString(ConfigurationKeys.FLOW_GROUP_KEY);
       String jobName = jobDataMap.getString(ConfigurationKeys.JOB_NAME_KEY);
-      String flowId = jobDataMap.getString(ConfigurationKeys.FLOW_EXECUTION_ID_KEY);
+      String flowExecutionId = jobDataMap.getString(ConfigurationKeys.FLOW_EXECUTION_ID_KEY);
       DagActionStore.DagActionType dagActionType = (DagActionStore.DagActionType) jobDataMap.get(FLOW_ACTION_TYPE_KEY);
 
       log.info("DagProc reminder triggered for (flowGroup: " + flowGroup + ", flowName: " + flowName
-          + ", flowExecutionId: " + flowId + ", jobName: " + jobName +")");
+          + ", flowExecutionId: " + flowExecutionId + ", jobName: " + jobName + ", dagActionType: " + dagActionType + ")");
 
-      DagActionStore.DagAction dagAction = new DagActionStore.DagAction(flowGroup, flowName, flowId, jobName,
-          dagActionType);
+      DagActionStore.DagAction dagAction = new DagActionStore.DagAction(flowGroup, flowName, flowExecutionId, jobName, dagActionType);
+      //dagAction.setReminder(true);
```

Comment:
do we not always want to set the reminder flag for actions coming from the trigger (potentially other than the deadline actions depending on how we decide to implement)

option 1: set reminder flag for all events
- handle deadline events knowing when it's a reminder = True is when we enforce

option 2: set reminder flag for all events except deadline if there's some other job property such as "firstTimeFiring=True" 
- set firstTimeFiring=False so all subsequent deadline actions are known to be true "reminders"

Reply:
actually we have a better method:
pass a flag to `createReminderJobTrigger` when the reminder is created to denote whether it's actually a reminder or not so for deadline triggers the first time we are created we don't set this value then for any subsequent attempt we set reminder=True. when we create reminder for all other actions types we set reminder=True

Reply:
oh, i forgot to set reminder during refactoring
it should be 
`DagActionStore.DagAction dagAction = new DagActionStore.DagAction(flowGroup, flowName, flowExecutionId, jobName, dagActionType, true);`

Reply:
just simply using the above constructor, because it is already inside `ReminderJob`